docs/code-samples/settlement/daml/Scripts/Transfer.daml

```diff
@@ -52,164 +22,100 @@ data SetupTransferRequest = SetupTransferRequest
-          Anyone -> S.empty
-      in
-        Account.Controllers with incoming = toParties incoming; outgoing = toParties outgoing
-    controllersWithOwner = toControllers accountControllers bank
-
-  -- Account Factory (it is used by the bank to create accounts)
-  accountFactoryCid <- toInterfaceContractId @Account.F <$> submit bank do
-    createCmd Account.Factory with provider = bank; observers = M.empty
-
-  -- Holding Factory (it is used by the bank to create holdings with the desired implementation)
-  holdingFactoryCid <- toInterfaceContractId @Holding.F <$> submit bank do
-    createCmd Fungible.Factory with
-      provider = bank
-      observers = M.fromList [("DisclosedToAllClients", S.fromList [alice, bob])]
-
-  -- Alice and Bob set up account @Bank
-  aliceRequestCid <- submit alice do
-    createCmd CreateAccount.Request with
-      custodian = bank
-      owner = alice
-      accountControllers = controllersWithOwner alice
-  aliceAccount <- submit bank do
-    exerciseCmd aliceRequestCid CreateAccount.Accept with
-      label = "Alice@Bank"
-      description = "Account of Alice at Bank"
-      accountFactoryCid
-      holdingFactoryCid
-      observers = []
-  bobRequestCid <- submit bob do
-    createCmd CreateAccount.Request with
-      custodian = bank
-      owner = bob
-      accountControllers = controllersWithOwner bob
-  bobAccount <- submit bank do
-    exerciseCmd bobRequestCid CreateAccount.Accept with
-      label = "Bob@Bank"
-      description = "Account of Bob at Bank"
-      accountFactoryCid
-      holdingFactoryCid
-      observers = [alice]
-
-  -- Bank creates the cash instrument
-  now <- getTime
-  let
-    cashInstrument = Instrument with
-      depository = bank
-      issuer = bank
-      id = Id "USD"
-      version = "0"
-      description = "Instrument representing units of USD"
-      validAsOf = now
-      observers = M.empty
-    instrument = Instrument.getKey $ toInterface cashInstrument
-  cashInstrumentCid <- toInterfaceContractId @Instrument.I <$> submit bank do
-    createCmd cashInstrument
-
-  -- Alice deposits cash at the bank
-  aliceRequestCid <- submit alice do
-    createCmd CreditAccount.Request with
-      account = aliceAccount
-      instrument
-      amount = 1000.0
-  aliceCashHoldingCid <- fromInterfaceContractId @Transferable.I <$> submit bank do
-    exerciseCmd aliceRequestCid CreditAccount.Accept
+  Setup{bank1 = bank, ..} <- runSetup accountControllers
 
   -- SETUP_TRANSFER_REQUEST_BEGIN
   let
     transferRequest = Transfer.Request with
       requestor
       receiverAccount = bobAccount
-      transferableCid = coerceContractId aliceCashHoldingCid
+      transferableCid = aliceHoldingCid
       accepted = S.fromList []
       observers = S.fromList [alice, bob, bank]
   transferRequestCid <- submit requestor do createCmd transferRequest
   -- SETUP_TRANSFER_REQUEST_END
 
-  pure SetupTransferRequest with ..
+  pure SetupTransferRequest{..}
```

Comment:
I would be consistent with the `..` notation, so either never use it (according to style guide) of use it with the `with` keyword

Reply:
I removed the .. notation, and made it explicit